src/reaction_diffusion_IO_routines.f90

```diff
@@ -82,7 +82,7 @@ SUBROUTINE REACTION_DIFFUSION_IO_WRITE_CMGUI(REGION, EQUATIONS_SET_GLOBAL_NUMBER
 
     !Argument variables
     TYPE(REGION_TYPE), INTENT(IN), POINTER :: REGION !<A pointer to the region to get the coordinate system for
-    CHARACTER(28), INTENT(IN) :: NAME !<the prefix name of file.
+    CHARACTER(30),INTENT(IN) :: NAME !<the prefix name of file.
```

Comment:
Whilst this IO routine is only temporary the CHARACTER length should be, for future reference, the MAXSTRLEN parameter.